Gemfile.lock

```diff
@@ -338,7 +338,7 @@ GEM
     zeitwerk (2.6.6)
 
 PLATFORMS
-  ruby
+  arm64-darwin-22
```

Comment:
Hmm I'm not sure if this will have any issues in any of the CircleCI machines...

Reply:
Yeah let me reverse this.